FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/scripts/mps2_m3.ld

```diff
@@ -75,6 +74,8 @@ SECTIONS
 		. = ALIGN(4);
 	} >FLASH
 
+    _sidata = LOADADDR(.data);
```

Comment:
Its a nit but it will be good if you can move this line to right before `.data` section.